README.md

```diff
@@ -22,9 +22,20 @@ yarn && yarn deploy
 
 The cli will prompt for approval on IAM Roles and Permissions twice in the full deploy. Once for the backend stack and then again for the client stack. The cli will prompt for an email. After the deploy is complete, an email will be sent to address provided with credentials for logging in.
 
+2. CICD Deploy:
+
+There is also a way to deploy the solution that invokes the same CICD pipeline that is used by Solutions Builder team. This will create a separate stack that loads all resource onto CodePipeline, and then uses CodePipeline to invoke CDK. Make sure stack->region inside package.json contains its original value, "%%REGION%%".
+
+```
+./deployment/build_and_deploy_project.sh [bucket-name-minus-region] [version] [email address]
```

Comment:
I believe this  instruction should be updated.
build_and_deploy_project.sh ->build-and-deploy-project.sh